web/src/app/administration/workbasket/details/information/workbasket-information.component.html

```diff
@@ -39,7 +39,12 @@ <h4 class="panel-header">{{workbasket.name}}&nbsp;
                 </div>
                 <div class="form-group required">
                     <label for="wb-owner" class="control-label">Owner</label>
-                    <taskana-type-ahead required #owner="ngModel" name="owner" [(ngModel)]="workbasket.owner" placeHolderMessage="Owner is required"></taskana-type-ahead>
+                    <taskana-type-ahead *ngIf="lookupField.visible else ownerInput" required #owner="ngModel" name="owner" [(ngModel)]="workbasket.owner"
+                        placeHolderMessage="Owner is required"></taskana-type-ahead>
```

Comment:
the access right owner is ment here !?

Reply:
What do you mean?

Reply:
never mind, i was not sure about the context here.. i think it is ok